root/static/scripts/common/components/AreaContainmentLink.js

```diff
@@ -14,7 +15,11 @@ import EntityLink from './EntityLink';
 
 const makeLink = (x, i) => <EntityLink entity={x} key={i} />;
 
-const AreaContainmentLink = ({area}) => (
+type Props = {
+  +area: AreaT,
+};
+
+const AreaContainmentLink = ({area}: Props) => (
```

Comment:
Wouldn't just `{area}: {area: AreaT}` be simpler here?

Reply:
It made the line too long once `: Expand2ReactOutput | null` was added in the last commit. I could've put the return type on a separate line, I guess, though I defaulted to making the Props type separate 'cause I think it makes adding new props a bit easer, and allows for reusing the type if we need to. But I don't mind either too much otherwise.